Geometry/HGCalTBCommonData/plugins/HGCalTBParametersESModule.cc

```diff
@@ -11,8 +11,6 @@
 #include "Geometry/HGCalTBCommonData/interface/HGCalTBParametersFromDD.h"
 #include "Geometry/Records/interface/IdealGeometryRecord.h"
 
-#define EDM_ML_DEBUG
```

Comment:
In this class in run time debug printout may be needed, so better to comment line 14 and not remove.

Reply:
@civanch , should I also remove `#ifdef ... #endif` around the `edm::LogVerbatim(..)` statements in this class?

Reply:
@smuzaffar , in this class debugging may be useful, so I would simply comment out line 14 and not remove it.

Reply:
done